RHEL/7/input/xccdf/system/software/updating.xml

```diff
@@ -118,6 +118,13 @@ or a yum server, run the following command to install updates:
 If the system is not configured to use one of these sources, updates (in the form of RPM packages)
 can be manually downloaded from the Red Hat Network and installed using <tt>rpm</tt>.
 </description>
+<!-- Directly define remotely located OVAL to be executed when scanning this rule -->
+<!-- Since OpenSCAP for now supports only "http://" protocol being specified when
+     fetching remote resources (see https://fedorahosted.org/openscap/ticket/213
+     for further details) use "http://" version of the remotely located OVAL below -->
+<check system="http://oval.mitre.org/XMLSchema/oval-definitions-5">
+  <check-content-ref href="http://www.redhat.com/security/data/oval/Red_Hat_Enterprise_Linux_7.xml" />
```

Comment:
Discussed this one with @iankko, if it works we would prefer to use the bzip2 variant here.